ios/Classes/TiMapSnapshotterProxy.m

```diff
@@ -12,6 +12,10 @@
 
 @implementation TiMapSnapshotterProxy
 
+CLLocationDegrees latitudeCoord;
+CLLocationDegrees longitudeCoord;
+bool showPin = NO;
```

Comment:
Please save the resulting `CLLocationCoordinate2D` instead, also use the ObjC `BOOL` type

Reply:
oh dear, this is old, I already forgot, I did this PR :-D
Ok, I will take care about this all next week ;-)